internal/services/compute/linux_virtual_machine_scale_set_resource.go

```diff
@@ -1447,4 +1447,43 @@ func resourceLinuxVirtualMachineScaleSetSchema() map[string]*pluginsdk.Schema {
 			Computed: true,
 		},
 	}
+
+	if !features.FivePointOh() {
+		schema["security_profile"].ConflictsWith = []string{
+			"encryption_at_host_enabled",
+			"secure_boot_enabled",
+			"vtpm_enabled",
+		}
+
+		schema["encryption_at_host_enabled"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeBool,
+			Optional:   true,
+			Deprecated: features.DeprecatedInFivePointOh("Use `security_profile` block instead."),
+			ConflictsWith: []string{
+				"security_profile",
+			},
+		}
+
+		schema["secure_boot_enabled"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeBool,
+			Optional:   true,
+			ForceNew:   true,
+			Deprecated: features.DeprecatedInFivePointOh("Use `security_profile` block instead."),
+			ConflictsWith: []string{
+				"security_profile",
+			},
+		}
+
+		schema["vtpm_enabled"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeBool,
+			Optional:   true,
+			ForceNew:   true,
+			Deprecated: features.DeprecatedInFivePointOh("Use `security_profile` block instead."),
```

Comment:
could we update this message to be inline with other deprecations?

Reply:
Done.